### PR TITLE
Restart Game

### PR DESCRIPTION
**Pull request by mentor-tara**
### Approve and merge this PR to start a new game.

This is the new board:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   |   |   |   |   |
| **3** |   |   |   |   |   |   |   |   |
| **4** |   |   |   |   |   |   |   |   |
| **5** |   |   |   |   |   |   |   |   |
| **6** |   |   |   |   |   |   |   |   |
| **7** |   |   |   |   |   |   |   |   |
| **8** |   |   |   |   |   |   |   |   |

[](restart-1)

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -1,8 +1,3 @@
 {
-  "commands": [
```
Comment (alantpetrescu) [inline commands.json:2]: Makes sense

**Comment by alantpetrescu** [comment]
hello there!

**Review by alantpetrescu — COMMENTED** [review]
Thanks!

**Review by alantpetrescu — APPROVED** [review]
Nice

File: commands.json [hunk 2]
```diff
@@ -1,8 +1,3 @@
...
-		"clear A8",
-		"flag C3",
-		"flag D2",
-		"Clear H8"
-	]
+  "commands": []
```
Comment (remmi755) [inline commands.json:2]: hello

File: commands.json [hunk 3]
```diff
@@ -1,8 +1,3 @@
 {
-  "commands": [
-		"clear A8",
-		"flag C3",
-		"flag D2",
```
Comment (remmi755) [inline commands.json:5]: gide

File: commands.json [hunk 4]
```diff
@@ -1,8 +1,3 @@
 {
-  "commands": [
-		"clear A8",
```
Comment (remmi755) [inline commands.json:3]: thanks

**Review by remmi755 — COMMENTED** [review]
thanks

File: commands.json (hunk 1 above)
Comment (remmi755) [inline commands.json:2]: hi

**Review by remmi755 — COMMENTED** [review]
gggg

**Pull request merged**